src/main/java/service/catalog/PriceCatalogVerticalImpl.java

```diff
@@ -0,0 +1,32 @@
+package service.catalog;
+
+import service.fileService.DataReader;
+import service.fileService.Property;
+
+import java.util.Map;
+
+public class PriceCatalogVerticalImpl implements PriceCatalogVertical {
+
+    private Map<Integer, Double> typeMap;
```

Comment:
What do you think about naming map with key and value? Something like priceByType.